elm-app/src/elm/Contact/View.elm

```diff
@@ -65,122 +77,161 @@ viewContacts baseUrl language showAsBlock { contacts, filterString } =
         if DictList.isEmpty filteredContacts then
             div [] [ text <| translate language ContactsNotFound ]
         else
-            ul
-                [ class "list list-primary list-borders" ]
-                (filteredContacts
-                    |> DictList.map
-                        (\contactId contact ->
-                            if showAsBlock then
-                                viewContactAsBlock baseUrl language ( contactId, contact )
-                            else
-                                viewContact baseUrl language ( contactId, contact )
+            viewMaybe
```

Comment:
@amitaibu Changed it to `viewMaybe`, just not sure about the 3rd argument if I did it correctly (syntax wise), also I'll create a follow up to change all the other `Utils.Html` functions.